src/python/WMCore/Storage/Backends/LCGImpl.py

```diff
@@ -105,7 +120,9 @@ def createStageOutCommand(self, sourcePFN, targetPFN, options=None, checksums=No
             result += "export LD_LIBRARY_PATH=`echo $LD_LIBRARY_PATH | sed -e 's+:*/cvmfs/cms.cern.ch/[^:]*++'g`\n"
 
             result += "echo Sourcing CVMFS UI setup script\n"
-            result += ". /cvmfs/grid.cern.ch/emi-ui-2.9.0-1_sl5v1/etc/profile.d/setup-cvmfs-ui.sh\n"
+            # this version dates back to Aug 2013. So, let's get the symlink to the latest one instead
+            #result += ". /cvmfs/grid.cern.ch/emi-ui-2.9.0-1_sl5v1/etc/profile.d/setup-cvmfs-ui.sh\n"
+            result += ". /cvmfs/grid.cern.ch/emi3ui-latest/etc/profile.d/setup-ui-example.sh\n"
```

Comment:
It's worth to mention that we were using a quite old EMI client. Updated it to the latest one in cvmfs, though I'm always afraid of pointing to the latest available version...